local-environment/dispatcher/conf/dispatcher/example.com.farm.any

```diff
@@ -0,0 +1,93 @@
+/example
+{
+  /clientheaders
+    {
+    "*"
+    }
+  /virtualhosts
+    {
+    "example.com"
```

Comment:
probably this value should come from `aem.publishInstances.first { it is LocalInstance }.properties["externalUrl"]`

consider adding predefined property / new class field `externalUrl`

Reply:
what is more that new property will be useful when integrating other features like AET / Functional tests which will require to have somewhere that external url.

If you will introduce it, I will consume it while integrating https://www.cypress.io/ into gradle-aem-multi

Reply:
In my opinion, this exceeds scope of this task.

Reply:
Okay, so that raise an issue when you complete this one